src/modules/differential_drive_control/DifferentialDriveControl.cpp

```diff
@@ -70,23 +90,45 @@ void DifferentialDriveControl::Run()
 	}
 
 	hrt_abstime now = hrt_absolute_time();
+	const double dt = static_cast<double>(math::min((now - _time_stamp_last), kTimeoutUs)) / 1e6;
+	_time_stamp_last = now;
 
 	if (_parameter_update_sub.updated()) {
-		parameter_update_s pupdate;
+		parameter_update_s pupdate{};
 		_parameter_update_sub.copy(&pupdate);
```

Comment:
```suggestion
		parameter_update_s parameter_update;
		_parameter_update_sub.copy(&parameter_update);
```
No need to zero initialize we anyways don't use the content afterwards.

Reply:
✅